File: shelper.php
<?php

class SocketHelper{
   
   public $socket;

   public function __construct($host,$port){
       $this->socket = socket_create(AF_INET,SOCK_STREAM,SOL_TCP);
   	   $result = socket_connect($this->socket,$host,$port);
   }

   public function send_data($content){
       socket_write($this->socket,$content,strlen($content));
   }

   public function read_data(){
   	   while($out = socket_read($this->socket,3)){
	       return $out;
       }
   }

   public function close_socket(){
   	   socket_close($this->socket);
   }
}

File: shelper.py
import socket

class SocketHelper:

	s = None
	conn = None
	addr = None

	def __init__(self,host,port):
		self.s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
		self.s.bind((host, port))
		self.s.listen(5)

	def s_appept(self):
		self.conn,self.addr = self.s.accept()

	def send_data(self,content):
		self.conn.send(content)

	def read_data(self):
		buf = self.conn.recv(1024)
		return buf

	def close_socket(self):
		self.conn.close()

